packages/repocop/src/metrics.ts

```diff
@@ -0,0 +1,86 @@
+import {
+	type CloudWatchClient,
+	type MetricDatum,
+	PutMetricDataCommand,
+} from '@aws-sdk/client-cloudwatch';
+import type { repocop_github_repository_rules } from '@prisma/client';
+import { getEnvOrThrow } from 'common/src/functions';
+
+export function getPercentageTrue(booleanishArray: Array<boolean | null>) {
+	const array = booleanishArray.filter((x) => x !== null) as boolean[];
```

Comment:
It might be a slightly more idiomatic to use a type guard ([some docs here](https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types)) here than an assertion, but up to you!

```suggestion
	const array = booleanishArray.filter((x): x is boolean => x !== null);
```